src/main/java/ru/rosatom/documentflow/dto/CountUsersDto.java

```diff
@@ -0,0 +1,16 @@
+package ru.rosatom.documentflow.dto;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor(force = true)
+@Schema(name = "Кол-во пользователей")
+public class CountUsersDto {
+
+    @Schema(name = "Количество пользователей")
```

Comment:
name - это ключ, имя переменной. Для описания следует использовать `description`
```suggestion
    @Schema(description = "Количество пользователей")
```